tests.bash

```diff
@@ -126,6 +126,14 @@ then
 
   killandsleep 8000
 
+  docker pull mongo:3.4.10
+  mkdir /tmp/mongodb
+  docker run -d -p 27018:27017 -v /tmp/mongodb:/data/db mongo:3.4.10
+  bash wait_for_local_port.bash 27018 40
+  S3BACKEND=mem MPU_TESTING=yes S3METADATA=mongodb npm start > $CIRCLE_ARTIFACTS/server_mongodb_awssdk.txt & bash wait_for_local_port.bash 8000 40 && npm run ft_awssdk
```

Comment:
It's just awssdk here, maybe you need to check this backend with the other test suits (s3cmd, s3curl, jaws, node) as well.

Reply:
Changing ft_awssdk by ft_test